config-editor/config-editor-ui/src/app/services/config-loader.service.ts

```diff
@@ -374,7 +375,10 @@ export class ConfigLoaderService {
         null
       )
       .pipe(map(result => {
-        if (!result.configs_files || (!result.test_cases_files && this.uiMetadata.testing.testCaseEnabled)) {
+        const testCaseIsEnabled = this.getTestSpecification().pipe(map( resp => {
+          return resp ? resp.find(x => x.name === DEFAULT_CONFIG_TESTER_NAME).test_case_testing : false;      
+        }));
+        if (!result.configs_files || (!result.test_cases_files && testCaseIsEnabled)) {
```

Comment:
testCaseIsENabled is an observable so should not work

Reply:
there might be a better way than calling the endpoint again, could be saved somewhere